data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -30,13 +30,42 @@ def get_survey_list(access_token: str) -> list:
 def get_survey_question_details(access_token: str, survey_id: str) -> list:
     headers = get_surveymonkey_api_headers(access_token)
     response = requests.get(
-        f'https://api.surveymonkey.com/v3/surveys/{survey_id}/details',
+        f"https://api.surveymonkey.com/v3/surveys/{survey_id}/details",
         headers=headers
     )
     reponse_json = response.json()
     return reponse_json
 
 
+def parse_answer_options_in_question_answer_json(
+    question_answer_response_json: dict
+) -> dict:
+    if isinstance(question_answer_response_json, list):
+        answer_options = question_answer_response_json
+    else:
+        answer_options = [question_answer_response_json]
+    return [
+        {
+            "id": answer_option.get("id", ""),
+            "text": answer_option.get("text", ""),
+            "type": answer_option.get("type", ""),
+            "weight": answer_option.get("weight", -99)
```

Comment:
Do we need the default value? Can't weight be None? Or don't include weight in the resulting object?
As this is a magic number that we would have to encode in SQL and would cause issues with aggregation functions if not handled specifically.

Reply:
(This links to https://github.com/elifesciences/data-hub-core-airflow-dags/pull/789#discussion_r761809544 which is now shown as outdated)

Reply:
Without default value I get creating schema problems, maybe we can look into thin in data catch-up